modules/admin_manual/pages/enterprise/user_management/user_auth_shibboleth.adoc

```diff
@@ -212,10 +212,20 @@ not known to the service provider.
 [[other-login-mechanisms]]
 === Other Login Mechanisms
 
-You can allow other login mechanisms (e.g. LDAP or ownCloud native) by
-creating a second Apache virtual host configuration. This second
-location is not protected by Shibboleth, and you can use your other
-ownCloud login mechanisms.
+You can allow other login mechanisms (e.g., LDAP or ownCloud native) by creating a second Apache virtual host configuration; such as in the below example.
+
+[source,apache]
+----
+include::example$enterprise/user_management/shibboleth/other-login-mechanisms-vhost.conf[]
```

Comment:
`include::example$` --> `include::{examplesdir}/`

In any case, I cant find the example file.
I used `find -type f -name '*conf'` in `modules/`
The only results were for ldap and nginx

Reply:
Seems I forgot to include it in the commit. `example$` is [the new format](https://docs.antora.org/antora/2.0/asciidoc/include-example/) for including examples. I believe it was introduced some months back.

Reply:
Just included the example file.

Reply:
Thanks for adding the file.
We currently have all includes to the examplesdir (plus partialsdir) plus in our best practices guide used curly brackets.
- This would breakt that and we then have two formats
- Any search for directives include examples would need then two searches.
Remember the mess with orphand examples

Either we use one directive or the other but not both.

Reply:
I'll update the old style to the new style in a separate PR.